doc/radosgw/layout.rst

```diff
@@ -21,32 +21,35 @@ Although RADOS only knows about pools and objects with their xattrs and
 omap[1], conceptually RGW organizes its data into three different kinds:
 metadata, bucket index, and data.
 
-* Metadata
+Metadata
+^^^^^^^^
 
 We have 3 'sections' of metadata: 'user', 'bucket', and 'bucket.instance'.
-You can use the following commands to introspect metadata entries:
+You can use the following commands to introspect metadata entries:::
```

Comment:
the rendered result looks like

You can use the following commands to introspect metadata entries::

could you please just put
```
You can use the following commands to introspect metadata entries::
```